app/assets/stylesheets/partials/_index.scss

```diff
@@ -80,6 +93,10 @@
     color: #fff;
     text-shadow: 0 -1px 0 rgba(0,0,0,.1);
     font-size: 13px;
+
+    &.--blue {
+      background: linear-gradient(#4ab6f0,#2fa5e4 60%,#2fa5e4 90%,#38a9e5);
```

Comment:
Color literals like `#4ab6f0` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Commas in function arguments should be followed by one space<br>Color literals like `#2fa5e4` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#38a9e5` should only be used in variable declarations; they should be referred to via variable everywhere else.